eng/SourceBuild.props

```diff
@@ -11,7 +11,8 @@
   -->
   <Target Name="ConfigureInnerBuildArg" BeforeTargets="GetSourceBuildCommandConfiguration">
     <PropertyGroup>
-      <InnerBuildArgs>$(InnerBuildArgs) /p:Projects="$(InnerSourceBuildRepoRoot)\Roslyn.sln"</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(DotNetBuildFromSource)' != 'true'">$(InnerBuildArgs) /p:Projects="$(InnerSourceBuildRepoRoot)\Roslyn.sln"</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(DotNetBuildFromSource)' == 'true'">$(InnerBuildArgs) /p:Projects="$(InnerSourceBuildRepoRoot)\Roslyn.SourceBuild.slnf"</InnerBuildArgs>
```

Comment:
😕 Why would **SourceBuild.props** be included in non-sourcebuild builds? Can we just make the `<Target>` element itself conditional?